2024-11-28 19:40:38,756 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-28 19:40:57,750 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-28 19:59:25,166 ERROR: Unhandled exception: Entity namespace for "campaign" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-28 23:34:59,775 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 00:00:30,448 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 00:01:44,598 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 00:10:22,876 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 00:10:32,203 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 00:25:39,053 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 09:11:00,975 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 09:11:12,406 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 09:27:25,179 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 09:27:37,337 ERROR: Error generando reporte de ventas: name 'get_stockouts' is not defined [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\facturas\facturacion_routes.py:283]
2024-11-29 09:28:01,498 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 09:30:58,405 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 09:31:38,457 ERROR: Error generando reporte: name 'get_stockouts' is not defined [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\facturas\facturacion_routes.py:286]
2024-11-29 11:51:05,029 ERROR: Error generando reporte: name 'get_stockouts' is not defined [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\facturas\facturacion_routes.py:286]
2024-11-29 12:26:04,380 ERROR: Unhandled exception: block 'content' defined twice [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:26:59,389 ERROR: Unhandled exception: block 'content' defined twice [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,326 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,328 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,335 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,356 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,394 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,455 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,479 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,481 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,486 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,527 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,529 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 12:40:25,534 ERROR: Unhandled exception: 429 Too Many Requests: 50 per 1 hour [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-29 14:04:55,159 ERROR: Error generando reporte: name 'get_stockouts' is not defined [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\facturas\facturacion_routes.py:286]
2024-11-29 14:06:49,839 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 14:10:21,769 ERROR: Error generando reporte: name 'get_stockouts' is not defined [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\facturas\facturacion_routes.py:286]
2024-11-29 14:11:51,496 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 14:14:09,957 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 14:14:54,312 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 14:15:03,287 ERROR: Error generando reporte: 'list' object has no attribute 'items' [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\facturas\facturacion_routes.py:302]
2024-11-29 14:16:33,847 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 14:20:53,776 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 14:25:33,154 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 14:27:04,383 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 14:27:35,958 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 14:34:48,772 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 14:41:14,635 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 14:45:43,770 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 15:03:58,855 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 15:22:02,201 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 15:25:01,390 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 15:26:09,781 ERROR: Error generando reporte: 'list' object has no attribute 'items' [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\facturas\facturacion_routes.py:376]
2024-11-29 15:26:21,422 ERROR: Error generando reporte: 'list' object has no attribute 'items' [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\facturas\facturacion_routes.py:376]
2024-11-29 16:54:32,689 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 16:54:45,060 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 16:55:40,717 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 17:03:26,738 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 17:04:21,362 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 17:05:30,505 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 17:05:47,063 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 19:02:01,248 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 19:02:09,569 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 19:04:45,477 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 19:10:30,889 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-29 19:10:55,154 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-30 17:40:23,860 ERROR: Unhandled exception: Entity namespace for "cliente_cxc" has no property "estado" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:701]
2024-11-30 17:42:59,084 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-30 17:48:02,906 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-30 17:49:30,121 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-30 17:58:03,967 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-30 18:03:48,496 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-30 18:05:44,767 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-30 18:50:34,769 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-30 18:57:22,604 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-30 18:57:54,481 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-30 18:58:37,033 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
2024-11-30 19:01:47,015 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:594]
